src/main/java/com/marklogic/client/ext/file/DocumentFile.java

```diff
@@ -65,7 +64,7 @@ public DocumentWriteOperation.OperationType getOperationType() {
 		return DocumentWriteOperation.OperationType.DOCUMENT_WRITE;
 	}
 
-	public DocumentMetadataWriteHandle getMetadata() {
+	public DocumentMetadataHandle getMetadata() {
```

Comment:
Not sure why I had this returning the fairly useless abstract type; the value is always an instance of `DocumentMetadataHandle`. Needed this change so that `DefaultSchemasLoader` can determinate if a file is a TDE template or not based on collections.